test/utilities/workbench.ts

```diff
@@ -43,24 +44,25 @@ export async function showExplorerView(): Promise<void> {
 export async function zoom(
   zoomIn: 'In' | 'Out',
   zoomLevel: number,
-  wait: number = 1
+  wait: Duration = Duration.seconds(1)
 ): Promise<void> {
   await zoomReset(wait);
   for (let level = 0; level < zoomLevel; level++) {
     await executeQuickPick(`View: Zoom ${zoomIn}`, wait);
   }
 }
 
-export async function zoomReset(wait: number = 1): Promise<void> {
+export async function zoomReset(wait: Duration = Duration.seconds(1)): Promise<void> {
   await executeQuickPick('View: Reset Zoom', wait);
 }
 
-async function handlePredicateOrWait(predicateOrWait: PredicateWithTimeout | number, prompt: unknown) {
-  if (typeof predicateOrWait === 'number') {
-    if (predicateOrWait > 0) {
+async function handlePredicateOrWait(predicateOrWait: PredicateWithTimeout | Duration, prompt: unknown) {
+  log('handlePredicateOrWait')
+  if (isDuration(predicateOrWait)) {
+    if (predicateOrWait && predicateOrWait.milliseconds > 0) {
```

Comment:
```suggestion
    if (predicateOrWait?.milliseconds > 0) {
```

Reply:
The type assertion on line 62 guarantees predicateOrWait is not undefined, but is also of type Duration, so the safe navigation is not needed either.